Daily-notes/TIL-20180305-brownbag-python-sean.md

```diff
@@ -0,0 +1,350 @@
+# Python
+
+An introduction to Python, by @seanwiseman :sparkles:
+
+These are my notes from Sean's brown bag on Python for beginners, March 5 (Sean has reviewed these notes)
+
+Output: [a basic script](/Scripts/article.py) to call the eLife API for information about an article using the article ID
+
+## To start
+
+We need:
+
+* A package manager:
+ * npm [Install from web](https://www.npmjs.com/get-npm)
```

Comment:
Not sure why this is included here? 🤔 `npm` is [node's](https://nodejs.org/en/) package manager. If you were to refer to a package manager here it would be [`pip`](https://pypi.org/project/pip/), but it will automatically be installed with along with python in most cases.

Reply:
Ah I think I was looking at ways to install python on the cmd line. But it's easier from the web. I'll amend.